### ISSUE TITLE
Define flag regex per service instead of global

### ISSUE THREAD

**Issue by Fxr12042** [OpenAttackDefenseTools/tulip#21]
Not all CTF's maintain flag format over different services. It would be good to either define a global flag format or define format (based on regex) per service, preferably in configurations.py with the services definitions

**Comment by RickdeJager** [comment]
I agree this would be a useful feature to have, but imo it would be better to move the config to some central format that both the webserver and the importer can understand.

I'm quite busy at the moment, and with ICSC (and ECSC) around the corner it will be a while before this repo sees any new big features.  

To work around this shortcoming, what you could do is use the Suricata plugin to populate the tags. That way you can use all of the context that Suricata provides to create separate alert rules for each flag format
